HdmiCec/HdmiCec.json

```diff
@@ -130,7 +193,7 @@
             }
         },
         "setEnabled":{
-            "summary": "Enables or disables HDMI-CEC",
+            "summary": "Enables or disables HDMI-CEC.\n  \n### Event \n\n No Events",
```

Comment:
Enables or disables HDMI-CEC driveer

Reply:
Updated changes as per your review comments
Please refer changes in #2155